Tests/LibWeb/Layout/input/box-sizing-border-box-for-definite-sizes-without-layout.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html><html><head><style>
+      .button {
+        align-items: center;
+        background-color: orange;
+        display: inline-flex;
+        height: 30px;
+        padding: 20px;
+        font-size: 20px;
+        box-sizing: border-box;
+        font: 20px SerenitySans;
```

Comment:
I guess `SerenitySans` needs to be specified for everything, not just `.button`. probably CI failed because there is a line inside body that got metrics computed with default font. might be wrong.

Reply:
Whoops yeah you're right!